ovisbot/extensions/utils/utils.py

```diff
@@ -61,6 +62,39 @@ async def rotn(self, ctx, shift, *params):
         shifted_str = msg.translate(shifted_tab)
         await ctx.send(f"{msg} => {shifted_str}")
 
+    @utils.command()
+    async def genshadow(self, ctx, *params):
+        argc = len(params)            
+        if argc == 1 or argc == 2:
+            cleartext = params[0]
```

Comment:
Can cleartext have whitespaces? If so how are you going to distinguish where it ends? If not you can just add two params in the function so that you don't have to handle the parameter handling + they will be displayed in the help text.